packages/core/src/internal/execution/types/execution-state.ts

```diff
@@ -42,6 +42,12 @@ export enum ExecutionSateType {
   SEND_DATA_EXECUTION_STATE = "SEND_DATA_EXECUTION_STATE",
 }
 
+/**
+ * The Strategy configuration that the user has provided in the run
+ * the future is started. Used to reconcile the future on subsequent runs.
+ */
+export type PassedStrategyConfig = Record<string, number | string>;
```

Comment:
How is it that we can have this narrower type here and a more generic one everywhere else? Or am I missing something?

Reply:
Maybe a better name than "Passed" could be "ConcreteExecutionConfig" or something like that?

Reply:
Updated to `ConcreteExecutionConfig`